tests/src/readytorun/crossgen2/crossgen2smoke.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>exe</OutputType>
+    <CLRTestKind>BuildAndRun</CLRTestKind>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <CLRTestPriority>0</CLRTestPriority>
+
+    <!-- Crossgen2 currently targets only x64 -->
+    <DisableProjectBuild Condition="'$(Platform)' != 'x64'">true</DisableProjectBuild>
+
+    <!-- Generate ILDLL so that the DLL can be the crossgenned assembly -->
+    <TargetExt>.ildll</TargetExt>
+
+    <!-- This is an explicit crossgen test -->
+    <CrossGenTest>false</CrossGenTest>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Program.cs" />
+  </ItemGroup>
+  <PropertyGroup>
+    <CLRTestBatchPreCommands><![CDATA[
+$(CLRTestBatchPreCommands)
+%Core_Root%\crossgen2\crossgen2 -r:%Core_Root%\*.dll -o:crossgen2smoke.dll crossgen2smoke.ildll
+]]></CLRTestBatchPreCommands>
+    <BashCLRTestPreCommands><![CDATA[
+$(BashCLRTestPreCommands)
+if [ -e "/etc/os-release" ]; then
+    source /etc/os-release
+fi
+if [ "${ID}" != "alpine" ]; then
```

Comment:
Can you please switch the crossgen2 TargetFramework to netcoreapp3.0 as I have found it fixes build of crossgen2 on Alpine and remove this check here?